crates/biome_js_parser/src/syntax/typescript/types.rs

```diff
@@ -189,9 +202,16 @@ pub(crate) fn parse_ts_type_parameters(p: &mut JsParser, context: TypeContext) -
 
     let m = p.start();
     p.bump(T![<]);
+
     if p.at(T![>]) {
-        p.error(expected_ts_type_parameter(p, p.cur_range()));
+        if context.is_in_type_or_interface_declaration() {
```

Comment:
I'd like to propose having only one variant,`TsTypeParameters`, for the type_parameters node.
It seems that we can try to write here:
```
if p.at(T![>]) && !context.is_in_type_or_interface_declaration() {
      p.error(expected_ts_type_parameter(p, p.cur_range()));
}
```

Reply:
What a simple solution! Thanks.